tools/ldb_cmd.cc

```diff
@@ -2948,8 +3072,23 @@ void DumpWalFile(Options options, std::string wal_file, bool print_header,
             break;
           }
         }
-        row << WriteBatchInternal::Sequence(&batch) << ",";
-        row << WriteBatchInternal::Count(&batch) << ",";
+        SequenceNumber sequence_number = WriteBatchInternal::Sequence(&batch);
+        uint32_t batch_count = WriteBatchInternal::Count(&batch);
+        assert(prev_batch_seqno);
+        assert(prev_batch_count);
+        // TODO(yuzhangyu): handle pessimistic transactions case.
+        if (only_print_seqno_gaps &&
+            (!prev_batch_seqno->has_value() || !prev_batch_count->has_value() ||
+             prev_batch_seqno->value() + prev_batch_count->value() ==
+                 sequence_number)) {
+          *prev_batch_seqno = sequence_number;
+          *prev_batch_count = batch_count;
+          continue;
+        }
+        row << sequence_number << ",";
+        row << batch_count << ",";
```

Comment:
Question: so what the new option does is print the batch after the gap, right? Would it make sense to print some info about the start of the gap (`prev_batch_seqno->value() + prev_batch_count->value()`) as well?

Reply:
Yes, that's a good point. I have updated it print a line like this:
Prev batch sequence number: 163356, prev batch count: 1, 164172,1,112,78,PUT(6) : 0x00000000000224C9000000000000012B00000000000000EF18E44F7BD5721300